modules/KoreBuild.Tasks/GenerateSignRequests.cs

```diff
@@ -136,18 +147,21 @@ internal bool Execute(Func<TextWriter> writerFactory)
                     {
                         if (!containers.TryGetValue(containerPath, out var container))
```

Comment:
What happens if there's an authoring error and something is listed as both an exclusion and an explicit signing item?

Reply:
Good point. Multiple items with the same file path is currently undefined behavior. We should add errors for that.